migration/src/lib.rs

```diff
@@ -25,6 +27,8 @@ impl MigratorTrait for Migrator {
             Box::new(m20230512_200213_make_task_max_batch_size_a_big_integer::Migration),
             Box::new(m20230512_202411_add_two_urls_to_every_task::Migration),
             Box::new(m20230616_223923_create_aggregators::Migration),
+            Box::new(m20230620_195535_add_aggregators_to_tasks::Migration),
+            Box::new(m20230622_232534_make_aggregator_api_url_mandatory::Migration),
```

Comment:
Since this is landing as one change, should this be one DB migration? I'm ok with keeping them as separate migrations since they're conceptually separate, or if the desire is that each migration affect a single table.

Reply:
I think they're fine as distinct migrations; this PR is effectively making those two unrelated changes that happened to get combined as one PR